.evergreen/config.in.yml

```diff
@@ -817,6 +817,40 @@ functions:
         args:
           - "${PROJECT_DIRECTORY}/.evergreen/run-snappy-version-test.sh"
 
+  "run lambda handler example tests":
+    - command: subprocess.exec
+      params:
+        working_dir: "src"
+        timeout_secs: 60
+        env:
+          MONGODB_URI: ${MONGODB_URI}
+          PROJECT_DIRECTORY: ${PROJECT_DIRECTORY}
+        binary: bash
+        args:
+          - "${PROJECT_DIRECTORY}/.evergreen/run-lambda-tests.sh"
+
+  "run lambda handler example tests with aws auth":
+    - command: shell.exec
+      type: test
+      params:
+        working_dir: "src"
+        silent: true
+        script: |
+          cd ${DRIVERS_TOOLS}/.evergreen/auth_aws
+          ${MONGODB_BINARIES}/mongo --verbose aws_e2e_regular_aws.js
+          cd -
+          cat <<EOF > "${PROJECT_DIRECTORY}/prepare_mongodb_aws.sh"
```

Comment:
Does this include the changes requested in [NODE-4585](https://jira.mongodb.org/browse/NODE-4585)?

Reply:
No. That ticket needs to make the change here: https://github.com/mongodb/node-mongodb-native/blob/034634165cf731498613d5d5474387125df16774/.evergreen/config.in.yml#L620